### ISSUE TITLE
_get_field_default_gen does not return the correct generator when freezegun is in use

### ISSUE THREAD

**Issue by markallanson** [danielgtaylor/python-betterproto#403]
The function `_get_field_default_gen` checks if a field's type is a datetime using `t is datetime`, which means if freezegun is being used in a test it generates the wrong default generator function.

Suggest this changes to `t instanceof datetime` to support cases like this.

**Comment by Gobot1234** [comment]
Fair enough, would you be willing to open a pull request to fix this?

**Comment by markallanson** [comment]
Yes! I am just heading out of the office on vacation and wanted to make sure I left at least a tombstone to create an PR for next week!
